docs/UserGuide.md

```diff
@@ -447,13 +468,16 @@ Edits a single calories goal if the goal exists.
 
 ## 🛌 Sleep Management
 
-Do note that that for sleep commands, the order of the parameters is fixed, and are all non optional.
+Do note that for sleep commands, the order of the parameters is fixed, and are all non-optional.
```

Comment:
Redundant information, we could delete that line (see Notes about Command Format, line 24)

Reply:
Yes good point made, deleted these lines.